packages/gatsby/cache-dir/ssr-builtin-trackers/tracking-unsafe-module-wrapper.js

```diff
@@ -1,42 +1,52 @@
 // initializing global here for unsafe builtin usage at import time
 global.unsafeBuiltinUsage = []
 
-function createProxyHandler(prefix) {
+function createProxyHandler(prefix, options) {
   return {
     get: function (target, key) {
       const value = target[key]
+      const path = key && key.toString ? `${prefix}.${key.toString()}` : prefix
```

Comment:
Just curious - is it possible to have falsy key here? Previously we just used `${prefix}.${key}`. Is it unsafe?

Reply:
not sure about falsy `key` - it was added just for complete check, but some key didn't have `.toString()` - for example `graceful-fs` use Symbol as a key to check if it patched `fs` yet ( https://unpkg.com/browse/graceful-fs@4.2.6/graceful-fs.js )- checks were already done in `if (typeof value === `object` && value !== null)` and that worked for `graceful-fs`, but probably better to have some checks for all code paths and not just one ;)